sesman.el

```diff
@@ -319,6 +332,16 @@ If SORT is non-nil, sort in relevance order."
 (defun sesman--lnk-value (lnk)
   (nth 2 lnk))
 
+(defun sesman--follow-symlink-p (filename)
+  "FILENAME predicate that tries to predict whether `find-file' will
```

Comment:
I am afraid this doesn't do it.  With default settings, If am about to open a symlink into a VCed project from outside the VC context I am redirected into the real file, but `sesman--follow-symlink-p` returns nil on that symlink.